content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -46,6 +46,14 @@ localization:
 
 .. _india/e-invoicing:
 
+Indian Configuration
+====================
+
+In :menuselection:`Settings --> Users & Companies --> Companies`, add your PAN and GST number.
```

Comment:
```suggestion
In :menuselection:`Settings --> Users & Companies --> Companies`, add your :guilabel:`PAN` and
:guilabel:`GSTIN` to ensure the correct chart of accounts and taxes are installed.
```

Reply:
We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

Reply:
> We use guilabels when talking about an interactive element such as a field to complete, and I suggested a little more information to give the reader more context as to why it's important.

Hello @dade-odoo , I have updated the information with correct reason for readers understanding.
thanks!!